README.md

```diff
@@ -20,25 +20,25 @@ Interested in contributing to this plugin? Feel free to join us in the [#core-si
 
 ### How can I fully disable sitemap generation?
 
-You can use `remove_action( 'init', 'core_sitemaps_get_server' );` to disable initialization of any sitemap functionality.
+You can use `remove_action( 'init', 'wp_sitemaps_get_server' );` to disable initialization of any sitemap functionality.
 
 ### How can I disable sitemaps for a certain object type?
 
-You can use the `core_sitemaps_register_providers` filter to disable sitemap generation for posts, users, or taxonomies.
+You can use the `sitemaps_register_providers` filter to disable sitemap generation for posts, users, or taxonomies.
```

Comment:
All these hooks should probably start with `wp_sitemaps_*`.

Reply:
👍 will correct.

Reply:
Fixed.